src/sentry/sentry_metrics/multiprocess.py

```diff
@@ -536,22 +538,59 @@ def __init__(
         self.__producer_topic = settings.KAFKA_TOPICS[settings.KAFKA_SNUBA_METRICS].get(
             "topic", "snuba-metrics"
         )
+        self.__commit_function = commit_function
 
         self.__closed = False
         self.__metrics = get_metrics()
+        self.__produced_message_offsets = {}
+        self.__callbacks = 0
+        self.__started = time.time()
+        # Need to make these flags
+        self.__commit_max_batch_size = 10000
+        self.__commit_max_batch_time = 4.0
+        self.__producer_queue_max_size = 8000
```

Comment:
I was just testing this at 8000 to see if I would enter in the long poll time if block